utils/prismicToolbarQueries.js

```diff
@@ -0,0 +1,22 @@
+import { Client } from 'utils/prismicHelpers';
+
+export const pageToolbarDocs = (uid, ref = null, lang) => (async () => {
+  const page = await Client().getByUID('page', uid, {ref, lang, fetch: 'page.display_title'})
+  const menu = await Client().getSingle('top_menu', { ref, lang, fetch: 'top_menu.display_title'}) 
```

Comment:
Is there a reason why you use getByUID here instead getSingle like the one below?

Reply:
Because there may be more than one doc for this Custom type, it's not a singleton